src/home/fetch-github/fetch-issues-preview.ts

```diff
@@ -79,12 +81,19 @@ export async function fetchIssuePreviews(): Promise<TaskNoFull[]> {
   } catch (error) {
     if (403 === error.status) {
       console.error(`GitHub API rate limit exceeded.`);
+      const resetTime = error.response.headers["x-ratelimit-reset"];
+      const resetParsed = new Date(resetTime * 1000).toLocaleTimeString();
+
       if (taskManager.getTasks().length == 0) {
-        // automatically login if there are no issues loaded
-        automaticLogin(error);
+        if (!user || user === null) {
+          // only show rate limit modal if there are no issues loaded and not logged in
```

Comment:
I think we should still show the modal even if only some of the issues are loaded.